codebase/superdarn/src.lib/tk/fitacf_v3.0/src/determinations.h

```diff
@@ -46,13 +49,63 @@ struct FitRange* new_range_array(FITPRMS* fit_prms);
 void ACF_Determinations(llist ranges, FITPRMS* fit_prms,struct FitData* fit_data,double noise_pwr);
 
 void lag_0_pwr_in_dB(struct FitRange* fit_range_array,FITPRMS* fit_prms,double noise_pwr);
+
+/*
+ * Setting the quality flag in fitted data. This determines if the data is good or bad. 
+ *
+ * @param fit_range_array: list of records from RAWACF 
+ * @param fit_prms: Fitacf parameters
+ * @param noise_pwr: noise power in dB
```

Comment:
`noise_pwr` has arbitrary units

```suggestion
 * @param noise_pwr: noise power
```